test/test-coordinator.js

```diff
@@ -316,4 +316,9 @@ TestCoordinator.prototype.getSUTHostPort = function getSUTHostPort() {
     return this.sutHostPort;
 };
 
+TestCoordinator.prototype.checksum = function checksum() {
+    //default to no-checksum. This method is changed based on runtime data of the SUT
+    return 0;
```

Comment:
Raise an exception isn't possible since the SUT will already start joining the fakeNodes before we had a chance to detect the checksum method. I'll change it to `null` to be more clear.